Source/Core/Core/IOS/Network/KD/Mail/MailParser.h

```diff
@@ -0,0 +1,106 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "MultipartParser.h"
```

Comment:
```suggestion
#include <MultipartParser.h>
```

Because this is an external